packages/react-scripts/backpack-addons/cssModules.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-// eslint-disable-next-line no-unused-vars
-const postcssNormalize = require('postcss-normalize');
 const paths = require('../config/paths');
 const appPackageJson = require(paths.appPackageJson);
 const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
 const cssModulesEnabled = bpkReactScriptsConfig.cssModules !== false;
+const noBackpackStylePrefixes =
+  bpkReactScriptsConfig.danger_noBackpackStylePrefixes !== false;
```

Comment:
Early tests show that we can live without these. Now I'd like to do more tests under this flag